src/components/SectionProduct/SectionProduct.jsx

```diff
@@ -1,36 +1,38 @@
-import { useQuery, gql } from '@apollo/client';
 import PropTypes from 'prop-types';
-import { CategoryContainer, Title, CardContainer } from './SectionProduct.styled.js';
+import { useQuery, gql } from '@apollo/client';
+
 import CardProduct from '../CardProduct/CardProduct.jsx';
 
+import { PRODUCT_LIST } from '../../constants';
+
+import { CategoryContainer, Title, CardContainer } from './SectionProduct.styled.js';
+
 export const SectionProduct = ({ title, request }) => {
   const {
     loading,
     error,
-    data: { getAllAdvertisement } = { getAllAdvertisement: [] },
+    data: { getAllAdvertisement: cardProductData } = { getAllAdvertisement: [] },
   } = useQuery(gql(request));
 
+  // TODO: Це тимчасово, поки бекенд не буде повертати досить даних, для їх використання
+  console.log(cardProductData);
+
   return (
     <CategoryContainer>
       <Title>{title}</Title>
 
       <CardContainer>
         {loading && <p>Loading...</p>}
 
-        {!loading && !getAllAdvertisement.length && <p>Немає оголошень, вийди розбійник</p>}
-
-        {!!getAllAdvertisement.length &&
-          Array.from({ length: 10 }).map((_, index) => (
-            <CardProduct
-              key={index}
-              images={getAllAdvertisement[0].images}
-              name={getAllAdvertisement[0].name}
-              location={getAllAdvertisement[0].location}
-              price={getAllAdvertisement[0].price}
-            />
+        {!error && !loading && !cardProductData.length && <p>Немає оголошень, вийди розбійник</p>}
```

Comment:
If you use the long conditions - you can create a special variable with these conditions before return and use it in JSX then